### PR TITLE
Shuttle collisions no longer run into limb damage caps on mobs

### PR DESCRIPTION
**Pull request by harmonyn**
Fixes https://github.com/tgstation/-tg-station/issues/14778.

Broken down:
- For human mobs, shuttle collisions now deal 60 damage to the chest and 60 damage to the head. The damage has been increased from 80 to 120 total in this case to account for likely pre-existing brute damage to either limb. Even 120 is really low for being hit by a fucking shuttle IMO, I'd be happy with shuttle gibbing returning, but that is a fair bit more controversial.
- Non-human mobs are gibbed outright.

### CODE REVIEW

**Comment by Niknakflak** [comment]
bring back shuttle gibbing 2016 :US: :US:

File: code/modules/shuttle/shuttle.dm [hunk 1]
```diff
@@ -455,12 +455,15 @@
...
+				if(istype(AM, /mob/living/carbon/human))
 					var/mob/living/M = AM
 					M.Paralyse(10)
-					M.take_organ_damage(80)
+					M.apply_damage(60, BRUTE, "chest")
+					M.apply_damage(60, BRUTE, "head")
 					M.anchored = 0
 				else
+					var/mob/M = AM
+					M.gib()
```
Comment (ChangelingRain) [inline code/modules/shuttle/shuttle.dm]: Looks like I need to override revenant gib to prevent unrevealed gibbing

Comment (bgobandit) [inline code/modules/shuttle/shuttle.dm]: how does revenant gib work

File: code/modules/shuttle/shuttle.dm [hunk 2]
```diff
@@ -455,12 +455,15 @@
 	for(var/turf/T in L)
 		for(var/atom/movable/AM in T)
 			if(ismob(AM))
-				if(istype(AM, /mob/living))
+				if(istype(AM, /mob/living/carbon/human))
```
Comment (phil235) [inline code/modules/shuttle/shuttle.dm]: ishuman(AM))

File: code/modules/shuttle/shuttle.dm (hunk 1 above)
Comment (bgobandit) [inline code/modules/shuttle/shuttle.dm]: @ChangelingRain

Comment (ChangelingRain) [inline code/modules/shuttle/shuttle.dm]: Actually I already did it and forgot don't worry about it

**Comment by bgobandit** [comment]
@phil235

File: code/modules/shuttle/shuttle.dm (hunk 1 above)
Comment (phil235) [inline code/modules/shuttle/shuttle.dm]: Are you sure M.gib() won't affect non living mobs? (/mob/)

Comment (bgobandit) [inline code/modules/shuttle/shuttle.dm]: are you asking me or @ChangelingRain

Comment (phil235) [inline code/modules/shuttle/shuttle.dm]: you

**Pull request merged**